printer_mods/README.md

```diff
@@ -153,6 +153,7 @@ like so:
 | jappaj | [V0.1 FanSaver](./jappaj/FanSaver) | A printable fix for overheating V0.1 blower fans | :heavy_check_mark: | :x: | :x: | :x: | :x:|
 |jasonc |[V2.4 Z-Idler Shoulder Bolt Mod](./jasonc/Z_Idler_Shoulder_Bolt) |Modification to the idler carriers to accept a 25mm shoulder bolt. Ideal for Gates idlers. |:x: |:x: |:heavy_check_mark: |:x:|
 | Jasrags | [Feed slot with filament pass-through](./Jasrags/sw_fee_slot_filament_passthrough) | A feed slot with a filament pass-through | :x: | :x: | :x: | :heavy_check_mark: |:x:|
+|JD |[V2.4 Filament runout sensor](./JD/RefillPlease) | Replace the 2.4 stock bowden tube holder and add a filament runout sensor  | :grey_question: | :grey_question: | :heavy_check_mark: | :grey_question: | :grey_question: |
```

Comment:
I think you can put an :x: for the v0 since the extrusions have different slot widths. Not sure where exactly you would put it on a switchwire, but it should definitely fit if one wanted to use it.

Reply:
Fixed for the V0 Thanks !

Reply:
Sample klipper configuration added + doc